docs/src/test/java/docs/http/javadsl/server/directives/BasicDirectivesExamplesTest.java

```diff
@@ -218,27 +218,18 @@ public void testWithSettings() {
     final RoutingSettings special =
       RoutingSettings
         .create(system().settings().config())
-        .withFileIODispatcher("special-io-dispatcher");
-
-    final Route sample = path("sample", () -> {
-      // internally uses the configured fileIODispatcher:
-      // ContentTypes.APPLICATION_JSON, source
-      final Source<ByteString, Object> source =
-        FileIO.fromPath(Paths.get("example.json"))
-          .mapMaterializedValue(completionStage -> (Object) completionStage);
-      return complete(
-        HttpResponse.create()
-          .withEntity(HttpEntities.create(ContentTypes.APPLICATION_JSON, source))
-      );
-    });
+        .withFileGetConditional(false);
```

Comment:
why this?

Reply:
as `withFileIODispatcher()` gets deprecated in this PR and it does not actually change settings, the example now demonstrates how to use `withSettings` just with another routing setting - `fileGetConditional` which looks to be a good replacement.